selenium_issoft/src/test/java/ByVariables.java

```diff
@@ -0,0 +1,22 @@
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class ByVariables {
+    @Test
+    void byVariables(){
+    System.setProperty("webdriver.chrome.driver", "d:\\Java\\issoft_github\\selenium\\selenium\\selenium_issoft\\drivers\\chromedriver.exe");
```

Comment:
please, think what for we stude java for 3-5 months, and OOP principles in paticular